### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by akabla**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -1,5 +1,5 @@
 using RHEOS
-using SpecialFunctions
+#using SpecialFunctions
```
Comment (moustachio-belvedere) [inline test/runtests.jl:2]: I think not exporting this will cause test failures?

File: src/interface.jl [hunk 2]
```diff
@@ -73,7 +75,7 @@ end
 
 
 function AFM(R::Real)
-   Interface(:d, :f, (ϵ,σ)->(d = (R*(3. /4.).^(2. /3)) .* ϵ.^(2. /3), f = R.^2 .* σ ), (d,f)->(ϵ = 4. /(3. *R^1.5) .* d.^1.5, σ = f./R^2) )
+   Interface(:d, :f, (ϵ,σ)->(d = (R*(3. /4.)^(2. /3)) .* ϵ.^(2. /3), f = R^2 .* σ ), (d,f)->(ϵ = 4. /(3. *R^1.5) .* d.^1.5, σ = f./R^2) )
```
Comment (moustachio-belvedere) [inline src/interface.jl]: Any reason why we `3. / 4. ` is better than just `3/4`? Also, why sometimes `x. / y.` and other times just `x. / y`?

File: docs/src/fitpredictFreq.jl [hunk 3]
```diff
@@ -51,7 +51,7 @@ FractKV_model = dynamicmodelfit(data, Fract_KelvinVoigt, weights = "log", lo = L
 
 data_ext = extract(data, freq_only)
 ## Alternatively, a RheoFreqData with only frequency data can be generated as
-## data_gen = frequencyspec(ω_start = 1.0e-2, ω_end = 1.0e2, step = 1.0e-5)
+## data_ext = frequencyspec(ω_start = 1.0e-2, ω_end = 1.0e2, logstep = 0.1)
```
Comment (moustachio-belvedere) [inline docs/src/fitpredictFreq.jl:54]: Good spot :+1:

**Review by moustachio-belvedere — COMMENTED** [review]
Was just curious what the PR was changing so added some questions. Looks good to me apart from the removal of specialfunctions import from runtests which may cause issues.

File: test/runtests.jl (hunk 1 above)
Comment (akabla) [reply test/runtests.jl:2]: Well, apparently not. So it is possible we don't need at after all. @ab2425 do you know why we have this requirement in the tests?

I first commented the line to resolve a local conflict but then realised it did work fine as it is too.

File: src/interface.jl (hunk 2 above)
Comment (akabla) [reply src/interface.jl]: I guess I'm a bit paranoid with making sure is it not interpreted as an integer division... It doesn't seem to be a problem in Julia though.

**Pull request merged**